presto-main/src/main/java/com/facebook/presto/util/MoreLists.java

```diff
@@ -49,5 +52,14 @@
                 .collect(toImmutableList());
     }
 
+    public static <T> List<T> nElements(int n, IntFunction<T> function)
```

Comment:
This one of these function that are being only "sometimes" reused even by the author itself. Someone who doesn't know that this very specific function particularly exist - never would try to lookup one, but instead will try to implement either a copy, or just inline the algorithm. I'm not even saying it is available only in a single module. I don't think it is worth adding such a functions.